docs/docs/docs/deprecated/_category_.yml

```
@@ -0,0 +1,5 @@
+position: 999
+label: "Deprecated Features"
+collapsible: true
+collapsed: true
+className: "sidebar-hidden" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)